src/config.ts

```diff
@@ -0,0 +1,74 @@
+const SERVICE_NAME = "auth-service";
+
+const {
+    NATS_URL = "nats://localhost:4222",
+    INTERNAL_REDIRECT_URI = "http://localhost:8000/auth/auth-service/authorize_code",
+    TOKEN_ENDPOINT = "",
+    USERINFO_ENDPOINT = "",
+    AUTHORIZE_ENDPOINT = "",
+    AUTHN_ENDPOINT = "",
+    SESSIONS_ENDPOINT = ""
+} = process.env;
+
+const HANDLERS_SUBJECTS = {
```

Comment:
this should be an enum

```
enum HANDLERS_SUBJECTS {
    TOKEN: "Token",
...
}
```

https://www.typescriptlang.org/docs/handbook/enums.html